packages/blade/src/components/Card/CardSurface.native.tsx

```diff
@@ -28,22 +26,16 @@ type CardSurfaceProps = {
   children: React.ReactNode;
   surfaceLevel: Exclude<SurfaceLevels, 1>;
   elevation: keyof Elevation;
-} & BaseBoxProps;
+} & Omit<BaseBoxProps, 'elevation'>;
 
 const CardSurface = ({
   children,
   surfaceLevel,
   elevation,
   ...props
 }: CardSurfaceProps): React.ReactElement => {
-  const { theme } = useTheme();
   return (
-    <CardSurfaceStyled
-      {...props}
-      surfaceLevel={surfaceLevel}
-      elevation={elevation}
```

Comment:
same why did we remove this from Card?

Reply:
Same as above, CardSurfaceStyled is extending BaseBox.